app/components/tag/_tag.scss

```diff
@@ -4,3 +4,13 @@
   font-weight: 600 !important;
   padding: nhsuk-spacing(1) nhsuk-spacing(3);
 }
+
+.bc-c-tag-outline {
+  color: $color_nhsuk-blue;
+  border: 2px solid $color_nhsuk-blue;
```

Comment:
use nhsuk-spacing()

Reply:
What if I want it to be 2px? nhsuk-spacing of 1 makes it 4px

Reply:
@kat1906 any suggestions on what Ben said above? or we good to roll with the hardcoded 2px?